src/isar/mission_planner/echo_planner.py

```diff
@@ -123,20 +120,15 @@ def _get_robot_pose(self, tag_name: str) -> Pose:
         predefined_pose: Pose = predefined_poses[tag_name]
 
         if predefined_pose.frame is Frame.Robot:
+            # TODO: Should we raise an error and require workaround pose to be asset frame?
```

Comment:
Should we raise and error and require the workaround poses to be asset frame?

Reply:
Yes.